static/cmd/main.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/status-im/status-go/geth/common/cipher"
+	"github.com/status-im/status-go/geth/params"
+	"gopkg.in/urfave/cli.v1"
+)
+
+/* Examples:
+go run main.go --input "../keys/wnodekey" --output="../keys/wnodekey.cr" --key="1234567891234567" encrypt
+go run main.go --input "../keys/wnodekey.cr" --output="../keys/wnodekey.crd" --key="1234567891234567" decrypt
+*/
+func main() {
+	if err := app.Run(os.Args); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+}
+
+var (
+	app = cli.NewApp()
+
+	keyFlag = cli.StringFlag{
+		Name:   "key",
+		EnvVar: "STATUS_KEY",
+		Usage:  "AES-128 or AES-256",
+	}
+
+	nonceFlag = cli.StringFlag{
+		Name:   "nonce",
+		EnvVar: "STATUS_NONCE",
+		Value:  params.DataDir,
+	}
+
+	inputFileFlag = cli.StringFlag{
+		Name:   "input",
+		EnvVar: "INPUT_FILE",
+		Usage:  "path to input file",
+	}
+
+	outputFileFlag = cli.StringFlag{
+		Name:   "output",
+		EnvVar: "OUTPUT_FILE",
+		Usage:  "path to output file",
+	}
+)
+
+var (
+	encryptCommand = cli.Command{
+		Action: EncryptCommandHandler,
+		Name:   "encrypt",
+		Usage:  "Print app version",
```

Comment:
Are you sure? :)

Reply:
Second this question - if there is a new tool introduced, let's put it into proper place and document properly. Also would be nice to write a few lines about it in PR description.

Also, there is in progress PR (I think it'll be done tomorrow or a couple of days more) that simplifies `statusd` command line interface, so it doesn't use `gopkg.in/urfave/cli.v1` anymore. You may want to use simpler approach with stdlib here as well.